pom.xml

```diff
@@ -0,0 +1,385 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.mesosphere.velocity</groupId>
+	<artifactId>velocity</artifactId>
+	<version>1.2-SNAPSHOT</version>
+
+	<repositories>
+		<repository>
+			<id>jenkins</id>
+			<url>http://maven.jenkins-ci.org/content/repositories/releases/</url>
+		</repository>
+	</repositories>
+	<dependencies>
+		<dependency>
+			<groupId>org.jenkins-ci.main</groupId>
+			<artifactId>jenkins-war</artifactId>
+			<version>1.658</version>
```

Comment:
Why is this 1.658 out of interest? Looks like the most recent stable was 1.651.3.